funnel/templates/macros.html.jinja2

```diff
@@ -99,6 +99,11 @@
 {% macro project_header(project, project_save_form, csrf_form, class='', current_page='project') %}
   <div class="mui-container project-banner {% if class %}{{ class }}{% endif %}">
     <div class="grid">
+      {% if current_auth.user.is_site_editor and project.featured %}
+        <div class="grid__col-sm-12">
+          {{ alertbox('success', 'This is a featured project') }}
```

Comment:
'This is now a featured project'

Reply:
Needs translation.

Reply:
I think `gettext` is available in this context (not `_`) so you can use that:

`{{ alertbox('success', gettext("This is now a featured project")) }}`